.gitignore

```diff
@@ -17,3 +17,5 @@ sophia
 # Vim Swap files
 *.sw[a-z]
 .idea
+
+venv/*
```

Comment:
Up to you:

`dev: ignore venv folder` -> `git: ignore venv folder`

Reply:
Fair, changed